### PR TITLE
auto DEM windows bug fix

### PR DESCRIPTION
**Pull request by bakkerbakker**
`pgc_ortho.py` and `pgc_pansharpen.py` were failing when run from Windows utilizing the `-d auto` option because the reference DEM was a `.vrt` that included linux paths instead of windows paths to reference the underlying DEMs within the `.vrt`. This patch points to a different version of the `.vrt` with windows-suitable paths.

### CODE REVIEW

File: lib/ortho_functions.py [hunk 1]
```diff
@@ -362,7 +362,6 @@ def convert_windows_path(path):
     if platform.system() == "Windows":
         # Replace "/mnt" with "V:"
         path = path.replace("/mnt", "V:")
-        path = path.replace("nunatak","windows")
         # Replace "/" with "\"
         path = path.replace("/", "\\")
```
Comment (clairecporter) [inline lib/ortho_functions.py]: This is not necessary.  Windows can handle forward slashes.

File: lib/ortho_functions.py [hunk 2]
```diff
@@ -530,6 +529,8 @@ def copy_to_wd(source_fp, wd):
             config.read(convert_windows_path(args.config_file))
             # Get the path from the config file
             gpkg_path = convert_windows_path(config.get("default", "gpkg_path"))
+            if platform.system() == "Windows" and "nunatak" in gpkg_path: # this is for selecting between specific versions of the reference DEM file
+                gpkg_path = gpkg_path.replace("nunatak","windows")
```
Comment (clairecporter) [inline lib/ortho_functions.py]: This sort of directory path hard-coding is fragile in a publicly used tool.  Consider a solution that puts the system-specific login in the geopackage itself, like having two attributes, one for linux and one for Windows.  Let me know if you want to discuss the different approaches you can take.

Comment (clairecporter) [reply lib/ortho_functions.py]: Also, shouldn't the [convert_windows_path](https://github.com/PolarGeospatialCenter/imagery_utils/blob/0a94f69e1298ae854d636bd01c2bb9b90b655ec0/lib/ortho_functions.py#L1955) function handle the dempath conversion without this?

Comment (bakkerbakker) [reply lib/ortho_functions.py]: The convert_windows_path function handles the path to the .vrt (the DEM that gets passed to the ortho function) but the paths defined _within_ the .vrt are still system dependent, so we need to point to the different DEM .vrt for windows.

**Pull request merged**